docs/ref/request-response.txt

```diff
@@ -473,20 +473,22 @@ a subclass of dictionary. Exceptions are outlined here:
 
 .. method:: QueryDict.items()
 
-    Just like the standard dictionary ``items()`` method, except this uses the
-    same last-value logic as ``__getitem__()``. For example::
+    Similar to the the standard dictionary ``items()`` method, except this uses the
+    same last-value logic as ``__getitem__()`` and returns an iterable
+    object instead of a view object. For example::
```

Comment:
Taking a look at the Python stdlib docs, it alternates between "return an iterator" or "make an iterator", it never says "yields". I changed this for "returns an iterator"